source/tutorials/hpc.rst

```diff
@@ -0,0 +1,693 @@
+.. _hpc:
+
+A Simple HPC Cluster on |CL|
+############################
+
+This tutorial demonstrates how to set a simple :abbr:`HPC (High
+Performance Computing)` cluster using `slurm`_, `munge`_, and 
+`PDSH`_.  For this tutorial, this cluster consists of a controller node 
+and four worker nodes, as shown in Figure 1.  For the sake of simplicity,
+each node resides on a separate host and their hostnames are hpc-controller,
+hpc-worker1, hpc-worker2, hpc-worker3, and hpc-worker4.  
+
+.. rst-class:: dropshadow
+
+.. figure:: /_figures/hpc/hpc-01.png
+   :alt: Simple HPC cluster
+
+   Figure 1: Simple HPC cluster
+
+The configuration is intentionally kept simple, notably avoiding setting 
+up cgroups and accounting. These and many more additional configuration 
+options can be added later.
+
+.. note::
+
+   This tutorial assumes you start with a new installation of |CL| for all 
+   nodes.
+
+Prerequisites
+*************
+
+* Knowledge and experience with HPC 
+* Familiarity with slurm, munge, and PDSH
+* All nodes have synchronized clocks (typically by NTP)
+
+.. contents::
+    :local:
+    :depth: 1
+
+Set up controller node
+**********************
+
+In this step, install the cluster tools, enable the slurm controller service, 
+and configure and enable the the munge service.
+
```

Comment:
The munge service should be installed/started before any other service (slurmctld, slurmd,..)
The Slurm systemd service unit files have : "After=network.target munge.service"

Reply:
Ok.  Order changed.